scripts/use-benchmark-results.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo Utilising benchmark results...
+mv *.rs "runtime/src/weights/"
```

Comment:
I this is all that's needed I think we can instead do `--output=runtime/src/weights` on the benchmark command

Reply:
That was the thing, yes. Not needed for this branch right now. But when merging 2.0, this would do a more usefull job. But if you change the benchmark command to do `--output=runtime/src/weights` then when merging 2.0, we need to change that script too.